mixer/pkg/api/grpcServer.go

```diff
@@ -131,6 +207,67 @@ func (s *grpcServer) Check(ctx context.Context, req *mixerpb.CheckRequest) (*mix
 	return resp, err
 }
 
+
+func (s *grpcServerEnvoy) checkEnvoy(ctx context.Context, req *authzGRPC.CheckRequest,
+	protoBag *attribute.EnvoyProtoBag, checkBag *attr.MutableBag) (*authzGRPC.CheckResponse, error) {
+	if err := s.dispatcher.Preprocess(ctx, protoBag, checkBag); err != nil {
+		err = fmt.Errorf("preprocessing attributes failed: %v", err)
+		lg.Errora("Check failed: ", err.Error())
+		return nil, grpc.Errorf(codes.Internal, err.Error())
+	}
+
+	lg.Debug("Dispatching to main adapters after running processors")
+	lg.Debuga("Attribute Bag: \n", checkBag)
+	lg.Debug("Dispatching Check")
+	// snapshot the state after we've called the APAs so that we can reuse it
+	// for every check + quota call.
+	//snapApa := protoBag.Snapshot()
+
+	cr, err := s.dispatcher.Check(ctx, checkBag)
+	if err != nil {
+		err = fmt.Errorf("performing check operation failed: %v", err)
+		lg.Errora("Check failed: ", err.Error())
```

Comment:
Here and elsewhere, should we use `ExtAuth.Check` instead of just `Check` to avoid confusion?

Reply:
Yes. That makes sense. I'll change that.